app/assets/config/manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_tree ../images
```

Comment:
This file shouldn't be necessary, I believe?

Reply:
The deploy fails on a manifest_backup step without it. I didn't look to deeply yet to see if we can skip that step.

Reply:
@aaron-collier See the "require what you need manually" section of https://github.com/capistrano/rails#usage